### PR TITLE
DiscoF4: Upgrade to ChibiOS

### PR DESCRIPTION
**Pull request by tracernz**
It boots and connects to GCS.

[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/179)

### CODE REVIEW

File: flight/targets/discoveryf4/fw/main.c [hunk 1]
```diff
@@ -69,6 +69,13 @@ int main()
 	/* Any new initialization functions should be added in OpenPilotInit() */
 	PIOS_heap_initialize_blocks();
 
+	#if defined(PIOS_INCLUDE_CHIBIOS)
```
Comment (mluessi) [inline flight/targets/discoveryf4/fw/main.c]: fix indetation

**Comment by mluessi** [comment]
LGTM otherwise

**Comment by mlyle** [comment]
Reviewed 9 of 9 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/d-ronin/dronin/179)*

**Comment by tracernz** [comment]
Review status: 8 of 9 files reviewed at latest revision, 1 unresolved discussion.

---

*[flight/targets/discoveryf4/fw/main.c, line 72 \[r1\]](https://reviewable.io:443/reviews/d-ronin/dronin/179#-K4vsXfFOPHtrrBg9v2L-r1-72:-K4z0rdnhgHcw0Fj3Xk0:-497468049) ([raw file](https://github.com/d-ronin/dronin/blob/2713f72ba5253d785d6f33827b8b8539f3bb3628/flight/targets/discoveryf4/fw/main.c#L72)):*
Fixed

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/d-ronin/dronin/179)*

**Comment by tracernz** [comment]
Review status: 8 of 9 files reviewed at latest revision, 1 unresolved discussion.

---

*[flight/targets/discoveryf4/fw/main.c, line 72 \[r1\]](https://reviewable.io:443/reviews/d-ronin/dronin/179#-K4vsXfFOPHtrrBg9v2L-r1-72:-K4z0wNpsbAenSGl78ui:-1346096948) ([raw file](https://github.com/d-ronin/dronin/blob/2713f72ba5253d785d6f33827b8b8539f3bb3628/flight/targets/discoveryf4/fw/main.c#L72)):*
Interesting to note that indentation is not clearly obvious on reviewable, that sucks since Github is really good.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/d-ronin/dronin/179)*

**Comment by mlyle** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/d-ronin/dronin/179)*

**Pull request merged**